main.lsf

```diff
@@ -7,335 +7,335 @@
+	Object src/field_use_item.o
+	Object asm/unk_020658D4.o
+	Object src/sys_flags.o
+	Object src/sys_vars.o
+	Object asm/unk_02066EDC.o
+	Object src/field_roamer.o
+	Object asm/unk_0206793C.o
+	Object asm/unk_02067A60.o
+	Object src/field_move.o
+	Object asm/unk_020689C8.o
+	Object src/gymmick_init.o
+	Object src/unk_02068F84.o
+	Object asm/unk_02068FC8.o
+	Object asm/unk_02069528.o
+	Object asm/unk_02069660.o
+	Object asm/unk_0206979C.o
+	Object src/follow_mon.o
+	Object src/map_preview_graphic.o
+	Object asm/field_take_photo.o
+	Object src/unk_0206B910.o
+	Object src/alph_checks.o
+	Object src/get_egg.o
+	Object src/unk_0206D494.o
+	Object src/pokemon.o
+	Object src/move.o
+	Object src/trainer_data.o
+	Object src/pokemon_storage_system.o
+	Object src/pokemon_icon_idx.o
+	Object src/party.o
+	Object src/pokedex_util.o
+	Object asm/unk_02074944.o
+	Object asm/unk_02074E5C.o
+	Object asm/unk_020755E8.o
+	Object asm/unk_020773AC.o
+	Object asm/unk_02077678.o
+	Object src/bag_view.o
+	Object src/item.o
+	Object src/bag.o
+	Object asm/unk_02078834.o
+	Object asm/unk_02078DD8.o
+	Object asm/unk_02078E30.o
+	Object asm/unk_0207CB7C.o
+	Object asm/unk_0207EB24.o
+	Object asm/unk_0207F42C.o
+	Object asm/unk_02080BB4.o
+	Object asm/unk_02082908.o
+	Object asm/unk_020850F4.o
+	Object asm/unk_02085604.o
+	Object asm/unk_020863F4.o
+	Object asm/unk_02087284.o
+	Object asm/unk_02087A78.o
+	Object asm/unk_02087E70.o
+	Object asm/unk_02087FD4.o
+	Object asm/unk_0208805C.o
+	Object asm/unk_02088288.o
+	Object asm/unk_0208B1AC.o
+	Object asm/unk_0208C3E4.o
+	Object asm/unk_0208DE40.o
+	Object src/ribbon.o
+	Object src/trainer_memo.o
+	Object asm/unk_0208F658.o
+	Object asm/unk_0208F814.o
+	Object asm/unk_0208FB64.o
+	Object src/use_item_on_mon.o
+	Object src/mail_misc.o
+	Object src/unk_02091054.o
+	Object asm/unk_020910D8.o
+	Object asm/unk_02091278.o
+	Object asm/unk_020912AC.o
+	Object src/dex_mon_measures.o
+	Object asm/unk_02091564.o
+	Object asm/unk_020915B0.o
+	Object asm/unk_02091664.o
+	Object src/move_relearner.o
+	Object asm/unk_02091880.o
+	Object asm/unk_02091CDC.o
+	Object asm/unk_020921A4.o
+	Object src/save_data_read_error.o
+	Object src/save_data_write_error.o
+	Object asm/unk_02092B04.o
+	Object asm/unk_02092BB8.o
+	Object src/unk_02092BE8.o
+	Object asm/unk_020931C4.o
+	Object src/unk_020932A4.o
+	Object asm/unk_020932E0.o
+	Object asm/unk_020957B0.o
+	Object asm/unk_02095DF4.o
+	Object src/phonebook_dat.o
+	Object src/choose_starter.o
+	Object asm/unk_020961D8.o
+	Object src/error_message_reset.o
+	Object src/sin_vcount.o
+	Object asm/unk_020965A4.o
+	Object asm/unk_020968B0.o
+	Object asm/unk_02096910.o
+	Object asm/unk_02096C88.o
+	Object asm/unk_02097024.o
+	Object asm/unk_020971F8.o
+	Object asm/unk_02097268.o
+	Object src/unk_02097720.o
+	Object asm/unk_020977CC.o
+	Object asm/unk_020979A8.o
+	Object asm/unk_02097B78.o
+	Object asm/unk_02097BE0.o
+	Object asm/unk_02097D3C.o
+	Object src/pokemon_mood.o
+	Object src/unk_02097F6C.o
```

Comment:
Why'd you flip spaces -> tabs in this file, and why only partially? Stay consistent pls

Reply:
must've been a bad merge